client/src/components/Banner/Banner.stories.js

```diff
@@ -193,4 +194,18 @@ storiesOf("Layout/Banner", module)
         placeholder="Search Trip"
       />
     </Banner>
+  ))
+  .add("Book Trip Banner without Text", () => (
+    <Banner 
+    bannerType= "BannerLarge"
```

Comment:
Could you make `bannerType`, `type`? That way it's more inline with the existing props. i.e. if you look at the `InputTextField` component, the prop is called `type`.

Reply:
updated bannerType to type.